pull-dependencies

```diff
@@ -11,13 +11,24 @@
 $version = '2.0'
 $isPublic = true
 
+# BEGIN_HIDE
```

Comment:
Should we have separate scripts for pulling dependencies from jamie and downloading from the NLP www server?